File: src/Services/InventoryService/InventoryService.Application/DTOs/Response/Warehouse/WarehouseResponseDto.cs
using InventoryService.Application.DTOs.Base;
using InventoryService.Application.DTOs.Response.WarehouseType;
using InventoryService.Domain.Enums;

namespace InventoryService.Application.DTOs.Response.Warehouse;

public class WarehouseResponseDto : BaseDto
{
    public string Name { get; set; }
    public WarehouseTypeResponseDto Type { get; set; }
    public string Location { get; set; }
    public Guid CompanyId { get; set; }
    public Guid ResponsiblePersonId { get; set; }
}

File: src/Client/inventory-track/src/app/models/dto/warehouse/warehouse-response-dto.ts
import { WarehouseType } from './enums/warehouse-type.enum';

export class WarehouseResponseDto {
  id!: string;
  name!: string;
  type!: WarehouseType;
  location!: string;
  companyId!: string;
  responsiblePersonId!: string;
}
